Packages/vcs/vcs/vector.py

```diff
@@ -660,6 +691,8 @@ def list(self):
         print "alignment = ", self.alignment
         print "type = ", self.type
         print "reference = ", self.reference
+        print "scaletype = ", self.scaletype
+        print "scalerange = ", self.scalerange
```

Comment:
@aashish24 I don't see the code to update the json, I dont think you need to update it (it should pick it up automagcally) but can you double check that json dump of a vector graphic method now contain these attributes as well? Probably worth adding a test. Actually are the json dump test still passing?